cloud/amazon/ec2_vpc_dhcp_options.py

```diff
@@ -75,6 +77,26 @@
         reset them to be empty.
     required: false
     default: false
+  tags:
+    description:
+      - Tags to be applied to a VPC options set if a new one is created, or
+        if the resource_id is provided. (options must match)
+    required: False
+    aliases: [ 'resource_tags']
```

Comment:
Per the module checklist, "If required is false you need to document default, even if the default is ‘null’ (which is the default if no parameter is supplied). Make sure default parameter in docs matches default parameter in code."

Can you add the default here as well as for the dhcp_options_id option?